templates/wechatgame/first-screen.js

```diff
@@ -190,6 +357,22 @@ function draw() {
     gl.enableVertexAttribArray(aTexCoord);
     gl.vertexAttribPointer(aTexCoord, 2, gl.FLOAT, false, vertexFormatLength * 4, vertexFormatLength * 2);
     gl.drawArrays(gl.TRIANGLE_STRIP, 0, 4);
+    // draw slogan
+    gl.useProgram(program);
+    gl.activeTexture(gl.TEXTURE0);
+    gl.bindTexture(gl.TEXTURE_2D, sloganTexture);
+    var uSampler = gl.getUniformLocation(program, 'u_Sampler');
+    gl.uniform1i(uSampler, 0);
+    gl.bindBuffer(gl.ARRAY_BUFFER, sloganVertexBuffer);
+    var vertexFormatLength = 4;
+    var aPosition = gl.getAttribLocation(program, 'a_Position');
+    gl.enableVertexAttribArray(aPosition);
+    gl.vertexAttribPointer(aPosition, 2, gl.FLOAT, false, vertexFormatLength * 4, 0);
+    var aTexCoord = gl.getAttribLocation(program, 'a_TexCoord');
+    gl.enableVertexAttribArray(aTexCoord);
+    gl.vertexAttribPointer(aTexCoord, 2, gl.FLOAT, false, vertexFormatLength * 4, vertexFormatLength * 2);
+    gl.drawArrays(gl.TRIANGLE_STRIP, 0, 4);
```

Comment:
It seems these codes are duplicated, can wrap it in a function?

Reply:
The attributes and shaders used in each section are not exactly the same. Like different between logo shader and prograss bar shader. Just because the shader of the logo and the background are roughly similar, it seems to be duplicated. So I think it is not necessary and unreasonable to encapsulate all of them into one function.

Reply:
done.